src/bin/bat/config.rs

```diff
@@ -16,8 +16,22 @@ pub fn config_file() -> PathBuf {
 }
 
 pub fn get_args_from_config_file() -> Result<Vec<OsString>, shell_words::ParseError> {
+    let mut config_file_path = vec![PathBuf::from("/etc/bat/config")];
+    config_file_path.push(default_config_path());
```

Comment:
so how does that work?

`/etc/bat/config` is always included but `~/.config/bat/config` might be overwritten by `BAT_CONFIG_PATH`? Is that a good idea?

Reply:
@sharkdp Thanks for the review. I'm thinking of moving the logic for getting path from env var `BAT_CONFIG_PATH` to a separate function (something like `get_config_file_path_from_env` and `default_config_path()` will just return `~/.config/bat/config`.